src/components/simulation/encounterResult.tsx

```diff
@@ -89,8 +107,28 @@ const TeamResults:FC<TeamPropType> = ({ round, team, stats }) => {
                                         </li>
                                     ))
 
-                                    if (!li.length) return <>No action</>
-                                    return li
+                                    //todo effects that disappear in the same round are not shown, which can be misleading
+                                    const buffCount = combattant.finalState.buffs.size
+                                    const bi = Array.from(combattant.finalState.buffs)
+                                        .map((buff, buffId) => (
+                                            (buffCount <= 3) ?
+                                                <li key={buffId}>
+                                                    <b>{buff[1].displayName}</b>{getBuffEffect(buff[1])}
+                                                </li> :
+                                                <>
+                                                    <b>{buff[1].displayName}</b>{(buffId < buffCount - 1) ? ',' : null}&nbsp;
+                                                </>
+                                    ))
```

Comment:
```suggestion
                                        .map(([buffId, buff], index) => (
                                            (buffCount <= 3) ?
                                                <li key={buffId}>
                                                    <b>{buff.displayName}</b>{getBuffEffect(buff)}
                                                </li> :
                                                <>
                                                    <b>{buff.displayName}</b>{(index < buffCount - 1) ? ', ' : null}
                                                </>
                                    ))
```

Also changed the `&nbsp;` because it was causing this issue:
![image](https://github.com/Trekiros/battleSim/assets/13800822/ba54cd27-25e8-4e0e-948a-c0afe59889ce)